tests/test_robustness.py

```diff
@@ -197,4 +201,13 @@ def test_add_slangify_typo(self) -> None:
         for sample in transformed_samples:
             self.assertNotEqual(sample.test_case, sample.original)
 
-            
+    def test_multipleperturbations(self) -> None:
+        """"""
```

Comment:
Would be great to add the same tests which inputs of different type than `SequenceClassificationSample`

Reply:
inside the  transform/__init__.py
```            if TestFactory.task in ('question-answering', 'summarization') and test_name != 'multiple_perturbations' :
                _ = [
                    sample.transform(test_func, params.get('parameters', {}))
                    if hasattr(sample, 'transform') else sample
                    for sample in data_handler_copy
                ]
                transformed_samples = data_handler_copy
```
here in 'question-answering' and  'summarization' we are not passing complete data at one go here we are passing data record by  record   to  transform method of  BaseQASample class inside the sample.py file 
```
    def transform(self, func, params:None,perturbations=None,**kwargs):
        
        if perturbations is None:
            sens = [self.original_question, self.original_context]
            self.perturbed_question, self.perturbed_context = func(sens, **params, **kwargs)
            self.category = func.__module__.split('.')[-1]

        else:
            sens = [self.original_question, self.original_context]

            self.perturbed_question, self.perturbed_context = func(sens,perturbations, params,**kwargs)
            self.category = func.__module__.split('.')[-1]
```


so if we want to add test for 'question-answering' and  'summarization'   inside the test_robustness.py
so the format will be
```
class RobustnessTestCase(unittest.TestCase):

    def setUp(self) -> None:
        self.multipleperturbations_qa =  [
            "20 euro note -- Until now there has been only one complete series of euro notes; however a new series, similar to the current one, is being released. The European Central Bank will, in due time, announce when banknotes from the first series lose legal tender status.",
            "is the first series 20 euro note still legal tender"
        ]



    def test_multipleperturbations_qa(self) -> None:
        """"""
        transformations = ["lowercase","add_ocr_typo","titlecase","number_to_word"]
       .
       .
       .
      transformed_samples=MultiplePerturbations.transform(self.multipleperturbations_qa,transformations,config=None)
        self.assertIsInstance(transformed_samples, list)

```

Reply:
I think this would be a nice addition and increase test coverage 😄